manifests/mod/ssl.pp

```diff
@@ -62,6 +62,9 @@
 #   - Debian/Ubuntu + Apache >= 2.4: 'default'.
 #   - Debian/Ubuntu + Apache < 2.4: 'file:${APACHE_RUN_DIR}/ssl_mutex'.
 #
+# @param ssl_reload_on_change
+#   Enable reloading of apache if the content of ssl files have changed.
```

Comment:
Perhaps this should clarify that it's only the SSL files that are configured here in mod_ssl, not those in vhosts.